README.md

```diff
@@ -1,5 +1,7 @@
 linux-system-roles/network
 ==========================
+[![Coverage Status](https://coveralls.io/repos/github/linux-system-roles/network/badge.svg)](https://coveralls.io/github/linux-system-roles/network)
+[![Travis Build Status](https://travis-ci.org/linux-system-roles/network.svg?branch=master)](https://travis-ci.org/linux-system-roles/network)
```

Comment:
I suggest to put the badges elsewhere because this does not look good when the source is used to generate user documentation in /usr/share/doc or similar.

Reply:
What would the other place be? All other projects I know show the badges there so that visitors can get a fast overview. AFAIK there is no other way to display them on the repo's main page. Checking /usr/share/doc, there are several packages that show the badges in README.md there, for example rpmlint, asciinema, criu, fwupd, neovim, pandoc, ... (checked on my Fedora 28 system).